test/e2e/apps/statefulset.go

```diff
@@ -871,6 +874,88 @@ var _ = SIGDescribe("StatefulSet", func() {
 		})
 	})
 
+	/*
+		Release: v1.19
+		Testname: StatefulSet rollover from stuck rollout
+		Description: When StatefulSet is updated with invalid image or gets similarly failing
+		on the first updated pod (highest ordinal) it should be able to update when the podSpec is corrected.
+	*/
+	framework.ConformanceIt("should rollover from stuck rollout", func() {
+		const (
+			rolloutTimeout = 5 * time.Minute
+		)
+		ctx := context.TODO()
+
+		statefulset := e2estatefulset.NewSimpleStatefulSet("pause", 2)
+		gomega.Expect(*statefulset.Spec.Replicas).To(gomega.BeEquivalentTo(2))
+		gomega.Expect(statefulset.Spec.Template.Spec.Containers).To(gomega.HaveLen(1))
+
+		ginkgo.By("Creating a statefulset")
+		statefulset, err := c.AppsV1().StatefulSets(ns).Create(ctx, statefulset, metav1.CreateOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

Comment:
verify: test-code says here needs to be replaced with `framework.ExpectNoError(err)`